src/satellite/src/satellite.js

```diff
@@ -66,22 +65,25 @@ class Satellite {
     this.server.listen(port, callback);
   }
 
-  stop(callback) {
+  stop() {
     const self = this;
```

Comment:
Let's not bother with this, or with `finish` below.  We don't need to set it to `null`.